control-plane-operator/controllers/hostedcontrolplane/snapshotcontroller/assets/04_serviceaccount-hypershift.yaml

```diff
@@ -0,0 +1,7 @@
+# *** AUTOMATICALLY GENERATED FILE - DO NOT EDIT ***
```

Comment:
how was this "AUTOMATICALLY GENERATED"?

Reply:
I think this is a result of copying stuff from https://github.com/openshift/cluster-csi-snapshot-controller-operator/pull/125/files#diff-4e43d482fba3e3ae505d0a6febd014e3b243c206c3f517c0d207d3c9d4ddeb8a . It was automatically generated from a base file in that repo and then copied.

Reply:
It's generated in the operator github repo and copied here. I added README.md to the asset dir. To see `manifests/*-hypershift` files in cluster-csi-snapshot-controller-operator repo, check https://github.com/openshift/cluster-csi-snapshot-controller-operator/pull/125.

Header "AUTOMATICALLY GENERATED FILE - DO NOT EDIT" is vendored from [build-machinery-go](https://github.com/openshift/build-machinery-go/blob/e25cf57ea46d9ce17de894b1a00dcf43ba12ee1a/make/targets/openshift/operator/profile-manifests.mk#L12) and it would need to be regenerated in too many repos.